subset/pentests/brute_client.py

```diff
@@ -0,0 +1,79 @@
+import socket, sys, json
+
+user_data=open('user_list.json')
+user_list = json.load(user_data)
+user_data.close()
+
+#username=['root','admin','manager']
+username= user_list['username'].split(",")
+
+#password=['root','pass','friend']
+password= user_list['password'].split(",")
+
+arguments = sys.argv
+
+print 'brute_client test begin'
+print 'address:' + arguments[1]
```

Comment:
nit -- these should be below the variable assignments, so it reads:
print 'address: ' + address

ideally, only reference "arguments[1]" in exactly one place -- makes the code a lot cleaner

Reply:
updated code to reflect, makes sense, dont want to waste precious processing resources